code/datums/helper_datums/getrev.dm

```diff
@@ -0,0 +1,77 @@
+/datum/getrev
```

Comment:
This may runtime on this code depending on if Kyet has us running in an isolated container or a full git repo. Check with him before doing this